Mailer/app/models/Repositories/LogsRepository.php

```diff
@@ -28,6 +28,13 @@ class LogsRepository extends Repository
         'suppress-unsubscribe' => 'hard_bounced_at',
     ];
 
+    public function all()
```

Comment:
We shouldn't probably allow (or suggest) selecting `all()` is a good idea. It might be millions of emails. Since CRM always needs this on per-user basis, let's change this to `allForEmail`.

This would also mean that you need to change `LogsHandler` and `LogCountsHandler` to have `email` required and rename them to `EmailLogsHandler` and `EmailLogCountsHandler` so it would be email address specific.

Reply:
That's why I used the maximum value of limit.

Reply:
This comment was made for the repository function. We don't want anybody (not even internally) to start request `all()` logs. We have 40G of data there and we don't want that to be triggered, not even accidentally.

And more I thought about it, the safer it was to me to just use "email logs" unless we really needed to expose "all logs" via API.

And as a bonus, we will be able to get rid of the 300 limit in the API schema.